lib/web/apps.go

```diff
@@ -86,15 +85,15 @@ func (h *Handler) clusterAppsGet(w http.ResponseWriter, r *http.Request, p httpr
 
 		app := server.GetApp()
 
-		ugs := make(types.UserGroups, len(app.GetUserGroups()))
-		for i, userGroupName := range app.GetUserGroups() {
+		ugs := types.UserGroups{}
```

Comment:
If user groups not being found is rare, might help to allocate the capacity of the slice all at once:

```suggestion
		ugs := make(types.UserGroups, 0, len(app.GetUserGroups())
```

Reply:
It could potentially be more common than I thought it would be, so I think I'll keep this as is.